### PR TITLE
Adding write_tsv functionality for Array[Array[Primitive]].  Closes #529

### LINKED ISSUE #529: Add write_tsv() support

### PR DESCRIPTION
**Pull request by scottfrazer**
Jose wants to use this function

* wdl4s PR: https://github.com/broadinstitute/wdl4s/pull/17
* Tyburn PR: https://github.com/broadinstitute/tyburn/pull/29
* Centaur PR: https://github.com/broadinstitute/centaur/pull/4

### CODE REVIEW

**Comment by scottfrazer** [comment]
WHEEL: @geoffjentry @mcovarr ... for simplicity can you also review the WDL4s and Tyburn PRs?  They're short!

**Comment by geoffjentry** [comment]
:+1:

File: src/main/scala/cromwell/engine/WdlStandardLibraryImpl.scala [hunk 1]
```diff
@@ -126,6 +126,7 @@ trait WdlStandardLibraryImpl extends WdlStandardLibraryFunctions {
   override def write_map(params: Seq[Try[WdlValue]]): Try[WdlFile] = writeToTsv(params, classOf[WdlMap])
   override def write_object(params: Seq[Try[WdlValue]]): Try[WdlFile] = writeToTsv(params, classOf[WdlObject])
   override def write_objects(params: Seq[Try[WdlValue]]): Try[WdlFile] = writeToTsv(params, classOf[WdlArray])
+  override def write_tsv(params: Seq[Try[WdlValue]]): Try[WdlFile] = writeToTsv(params, classOf[WdlArray])
```
Comment (cjllanwarne) [inline src/main/scala/cromwell/engine/WdlStandardLibraryImpl.scala]: so write_tsv is functionally identical to write_objects?

Comment (cjllanwarne) [inline src/main/scala/cromwell/engine/WdlStandardLibraryImpl.scala]: Oh, and write_object!?

Comment (cjllanwarne) [inline src/main/scala/cromwell/engine/WdlStandardLibraryImpl.scala]: And write_map...?

**Comment by cjllanwarne** [comment]
:+1:

File: src/main/scala/cromwell/engine/WdlStandardLibraryImpl.scala (hunk 1 above)
Comment (scottfrazer) [inline src/main/scala/cromwell/engine/WdlStandardLibraryImpl.scala]: heh yeah...  also `write_lines` uses `writeToTsv` too.

`write_lines` - 1 column TSV.
`write_map` - 2 column TSV.  Column 1 is key, column 2 is value
`write_object` - 2 row TSV.  Row 1 is is headers, row 2 are the corresponding values
`write_objects` - 2+ row TSV.  Row 1 is headers, row 2-n are the corresponding values for each object
`write_tsv` - Most general case of serializing any `Array[Array[Primitive]]` as a TSV

**Pull request merged**